File: 02_ProgrammingFundamentals/_10_TextProcessing/_01_Lecture/_05_DigitsLettersAndOther.java
package _02_ProgrammingFundamentals._10_TextProcessing._01_Lecture;

import java.util.Scanner;

public class _05_DigitsLettersAndOther {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        String console = scan.nextLine();

        // Add variables
        StringBuilder digits = new StringBuilder();
        StringBuilder letters = new StringBuilder();
        StringBuilder other = new StringBuilder();

        // Separate digits, letters and other
        for (int i = 0; i < console.length(); i++) {
            char symbol = console.charAt(i);

            if (Character.isDigit(symbol)) {
                digits.append(symbol);
            } else if (Character.isLetter(symbol)) {
                letters.append(symbol);
            } else {
                other.append(symbol);
            }
        }

        // Print result
        System.out.println(digits);
        System.out.println(letters);
        System.out.println(other);

        scan.close();
    }
}

File: 24_C#Fundamentals/_08_TextProcessing/_01_Lecture/_05_DigitsLettersAndOther/_05_DigitsLettersAndOther.cs
using System;
using System.Text;

public class _05_DigitsLettersAndOther
{
    public static void Main(string[] args)
    {
        // Read input value
        string console = Console.ReadLine();

        // Add variables
        StringBuilder digits = new StringBuilder();
        StringBuilder letters = new StringBuilder();
        StringBuilder other = new StringBuilder();

        // Separate digits, letters and other
        for (int i = 0; i < console.Count(); i++)
        {
            char symbol = console[i];

            if (char.IsDigit(symbol))
            {
                digits.Append(symbol);
            }
            else if (char.IsLetter(symbol))
            {
                letters.Append(symbol);
            }
            else
            {
                other.Append(symbol);
            }
        }

        // Print result
        Console.WriteLine(digits);
        Console.WriteLine(letters);
        Console.WriteLine(other);

    }
}
